rigex-front-end/src/App.js

```diff
@@ -0,0 +1,9 @@
+function App() {
```

Comment:
can you do me a favor and make this class based instead of functional

Reply:
Can do!